### ISSUE TITLE
Feature request: APP_SWITCH with middle-click

### ISSUE THREAD

**Issue by mmm4m5m** [Genymobile/scrcpy#1063]
I think, "APP_SWITCH" is more important than "HOME".
For example, if I want to drink beer and work with one hand, I need APP_SWITCH much more than HOME.

I would like to suggest a command line parameter for "APP_SWITCH with middle-click". (And keyboard for "HOME").

Currently I found one good (not bad) workaround for "APP_SWITCH using mouse".
From android settings I assigned "Recents" (APP_SWITCH) action to "double tap home". This way I have APP_SWITCH with "double middle-click".

I find it is a bit hard to do quick "double middle-click". Double left/right click is more easy. It depends of the mouse also. And android do not have setting to increase "double tap delay".
(It could be useful if I can slowly "double middle-click" BUT it is sent faster/shorter to device.)

Thanks!

**Comment by rom1v** [comment]
Thank you for your suggestion.

It seems to me that such an option would be too specific: why an option to configure only app_switch or home on middle click? Another user might want to map it to right click instead…

More generally, it's a feature request about keymapping/customization of shortcuts.

**Comment by mmm4m5m** [comment]
Yes, I described my case specifically. You/devs can see, find and decide more general solution. 

But as you said, there are only 2 mouse buttons and android have 3 buttons (home, back, app_switch). Looks like it enough reason. Maybe command line switch could be simple, easy and fast solution.

**Comment by prometx** [comment]
I am not sure I am taking the op's meaning correctly, but if "APP_SWITCH", refers to the android function that shows cascading tiles of all currently running apps, and offers individual dismissal, or "clear all" functionality; I *whole-heartedly* agree.

This app is virtually golden, but this should be its very next function addition...#imho...please...

Hear, hear; Bully Bully...

**Comment by rom1v** [comment]
What do you think about Ctrl+middle-click or something like that for APP_SWITCH?

**Comment by Peter-maximus** [comment]
completely agree to have the appswitch implemented by mouse trigger.
ctrl+midlle click would be also fine, as you can "blindly" find it on the keyboard with left hand and still have right hand on the mouse...
however current state of things of just keyboard shortcut available is not ergonomic. you need to definitely look into keyboard to find the respective keys...
